pacmacs.el

```diff
@@ -130,18 +131,58 @@
         :column column
         :type 'wall))
 
-(defun pacmacs--make-pill (row column anim-name points)
+(defun pacmacs--make-pill (row column anim-name points type)
```

Comment:
You have them already separated, they named `pacmacs--make-regular-pill` and `pacmacs--make-big-pill` and even are expressed in terms of this `pacmacs--make-pill` function. So yeah, take a look, but probably you won't need to change anything here. It is nice already.